service/src/main/resources/config/verily/sdd/original/entity/platform.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "platform",
+  "idAttribute": "id",
+  "attributes": [
+    { "type": "SIMPLE", "name": "id" },
+    { "type": "SIMPLE", "name": "name" }
+  ],
+
+  "sourceDataMapping": {
```

Comment:
In the SDD UI, this entity has a hierarchy. I think the parent-child relationships live in the `genotype_criteria` table. You can define a `hierarchyMapping` in this source mapping block.

Reply:
In the Wed meeting, I'll ask them if they want hierarchy. Regardless, I'll do in a separate PR.